openbsd/OpenBSDProcessList.c

```diff
@@ -95,16 +96,29 @@ void ProcessList_delete(ProcessList* this) {
 static inline void OpenBSDProcessList_scanMemoryInfo(ProcessList* pl) {
    static int uvmexp_mib[] = {CTL_VM, VM_UVMEXP};
    struct uvmexp uvmexp;
-   size_t size = sizeof(uvmexp);
+   size_t size_uvmexp = sizeof(uvmexp);
 
-   if (sysctl(uvmexp_mib, 2, &uvmexp, &size, NULL, 0) < 0) {
+   if (sysctl(uvmexp_mib, 2, &uvmexp, &size_uvmexp, NULL, 0) < 0) {
       err(1, "uvmexp sysctl call failed");
    }
 
    //kb_pagesize = uvmexp.pagesize / 1024;
-   pl->usedMem = uvmexp.active * pageSizeKb;
+   //pl->usedMem = uvmexp.active * pageSizeKb;
```

Comment:
No need to keep old code around as comments; we can always refer to version control to look at the old version. :)